va/va.go

```diff
@@ -604,6 +607,36 @@ func (va *ValidationAuthorityImpl) getTLSSNICerts(hostPort string, identifier co
 	return certs, nil
 }
 
+// tlsDial does the equivalent of tls.Dial, but obeying a context. Once
+// tls.DialContextWithDialer is available, switch to that.
```

Comment:
Is `tls.DialContextWithDialer` blocked on a specific Go release?

Reply:
There's an open Go issue, and an externally-contributed patch, but no commitment from the Go team to land it by a specific release.